hathor/p2p/rate_limiter.py

```diff
@@ -68,17 +68,19 @@ def add_hit(self, key: str, weight: int = 1) -> bool:
 
         :param weight: How many hits this 'hit' means
         """
+        assert self.reactor is not None
+
         if key not in self.keys:
             return True
         (max_hits, window_seconds) = self.keys[key]
 
         if key not in self.hits:
-            self.hits[key] = RateLimiterLimit(weight, self.reactor.seconds())
+            self.hits[key] = RateLimiterLimit(weight, int(self.reactor.seconds()))
```

Comment:
Why round down the time? I feel we can work with the float number here.

Reply:
It was annotated with `int` before. But I checked and it seems that I added the `int` annotation which is wrong. I'll change the annotation instead.

Reply:
Done.

Reply:
Great! Thanks!